### PR TITLE
Fixing CI Typo and Cycamore mentions

### PR DESCRIPTION
**Pull request by gtw2**
This PR addresses a `recyle` typo and switches mention from Cycamore to Recycle appropriately. In addition, 2 nosetest files for Cycamore were removed.

This will address @bam241's comment in #16 for a separate PR.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@ jobs:
...
             - image: cyclus/cyclus:latest
-        working_directory: ~/recyle
+        working_directory: ~/recycle
         steps:
             # Ensure your image has git (required by git to clone via SSH) so that CircleCI can clone your repo
             - run: apt-get -qq update; apt-get -y install git openssh-client
             - checkout
             - run:
-                name: Build Cycamore
+                name: Build Cyclus
```
Comment (bam241) [inline circle.yml]:
```suggestion
                name: Build Recycle
```

**Comment by bam241** [comment]
thx @gtw2 for putting this in!

LGTM! let's wait for the test :)

**Comment by gtw2** [comment]
I believe this will be a chicken and the egg type situation since there isn't a facility to build in the current recycle haha

**Comment by bam241** [comment]
ok I'll merge it, and we see what happend on the other side :)
thx @gtw2

**Pull request merged**